core-web/apps/dotcms-ui/src/app/portlets/dot-porlet-detail/dot-portlet-detail.component.spec.ts

```diff
@@ -138,6 +141,7 @@ describe('DotPortletDetailComponent', () => {
     it('should have dot-contentlets', () => {
         spyOnProperty(router, 'parent', 'get').and.returnValue({
             parent: {
+                // @ts-ignore
```

Comment:
We leave `lint` out in the tests, something in the migration might be remove that setting, we need it back.

Reply:
removed